app/assets/javascripts/admin/app/controllers/gobierto_attachments_controller.js

```diff
@@ -457,6 +478,10 @@ this.GobiertoAdmin.GobiertoAttachmentsController = (function() {
           self.attachments.splice(index, 1);
           self.showFiles = true;
         });
+      },
+      created: function () {
+        var self = this;
```

Comment:
why do you need the `created` callback and the `mounted` callback to fech the data? It shoudn't be necessary